source/MaterialXFormat/Util.h

```diff
@@ -37,18 +40,12 @@ void loadLibrary(const FilePath& file, DocumentPtr doc, const FileSearchPath* se
 
 /// Load all MaterialX files with given library names in given search paths.
 /// Note that all library files will have a URI set on them.
-StringVec loadLibraries(const StringVec& libraryNames,
+StringVec loadLibraries(const FilePathVec& libraryNames,
```

Comment:
Cleanup to have 1 interface for library load. Can be fed the results of subdirectory search. This is not recursive in itself.